tests/unit/utils/test_parsers.py

```diff
@@ -499,21 +499,20 @@ def test_log_created(self):
         log_file_name = self.logfile_config_setting_name
         opts = self.default_config.copy()
         opts.update({'log_file': log_file})
-        if log_file_name is not 'log_file':
-            opts.update({log_file_name:
-                         getattr(self, log_file_name)})
```

Comment:
I believe we still need to keep this getattr and keep it at `log_file_name` on line 516.. Essentially what this is doing is if `log_file_name` is not `log_file` it means we are expecting a different argument in the configs to set where we are going to log. For example for ssh it looks for `ssh_log_file` instead of `log_file`. So in the test we are grabbing that value with `getattr(self, log_file_name)` since log_file_name equals `ssh_log_file`. This ensures we are testing that we are creating the log file that we are setting intentionally in the config instead of the default. Hopefully that explanation makes sense and maybe there is a better way of doing it that you see. Thanks for catching the is part!!

Reply:
Good call on the attribute, I didn't grok what you were doing there and thought the log file being tested was the one being set to the local `log_file` variable. Thanks for the review!

Reply:
Looks great :) thanks